1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -5,19 +5,27 @@
 */
 
 const BIRTHDAYS = [
-    "January 7th",
-    "February 12th",
-    "April 3rd",
-    "April 5th",
-    "May 3rd",
-    "July 11th",
-    "July 17th",
-    "September 28th",
-    "November 15th"
+  "January 7th",
+  "February 12th",
+  "April 3rd",
+  "April 5th",
+  "May 3rd",
+  "July 11th",
+  "July 17th",
+  "September 28th",
+  "November 15th",
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  let i = 0;
+  let julyBirthdays = [];
+  while (birthdays[i]) {
+    if (birthdays[i].includes("July")) {
+      julyBirthdays.push(birthdays[i]);
```

Comment:
I like how you did this Saadia. I ended up having to define "July 11th" as the string I was searching for instead of finding the first string containing July. Nice work!